pipelines/ApVerifyWithFakes.yaml

```diff
@@ -52,6 +76,7 @@ tasks:
             connections.diffIm: fakes_deepDiff_differenceExp
             connections.diaSourceTable: fakes_deepDiff_diaSrcTable
             # TODO: end DM-30210 workaround
+    # Can't have separate with- and without-fakes runs for diaPipe, because there's only one APDB
```

Comment:
I'm trying to think of a way we could have more than one Apdb for runs like this. Perhaps we could make a pipeline task with SkyMap as the only quantum, re-use a lot of our make_apdb code and then write a simple marker that is is the config data for the Apdb? Perhaps as a InputInit for the downstream tasks?